libs/@blockprotocol/type-system/typescript/test/shared.ts

```diff
@@ -0,0 +1,22 @@
+import fs from "node:fs/promises";
```

Comment:
I think this will fix the lint error – you will probably need to make the same change to the two other places this appears in the repo when those packages are converted to ESM
```suggestion
import * as fs from "node:fs/promises";
```

Reply:
This sadly didn't help. I am guessing might have something to do with ts node vs. web?

Reply:
adding `@types/node` (which apparently wasn't yet) fixed the problem.